ds-controller-board/res/devices/device-list.yaml

```
@@ -0,0 +1,12 @@
+# Copyright © 2023 Intel Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Pre-define Devices
+deviceList:
+  dame: controller-board
+  profileName: ds-controller-board
+  protocols: 
+    autoEvents:
+      interval: 3s
+      onChange: true
```

Indentation in TOML we off . I matter in YAML, but not in TOML.
```suggestion
  protocols: 
  autoEvents:
    interval: 3s
    onChange: true
```